xmtp_mls/src/groups/intents.rs

```diff
@@ -101,9 +100,9 @@ impl AddMembersIntentData {
         data: &[u8],
         provider: &XmtpOpenMlsProvider,
     ) -> Result<Self, IntentError> {
-        let msg = AddMembersPublishData::decode(data)?;
+        let msg = AddMembersData::decode(data)?;
```

Comment:
I had renamed this message in the proto repo but never updated in `libxmtp`, so I got forced into doing it now.